setup.py

```diff
@@ -71,8 +69,7 @@ def parse_reqs(req_path="./requirements.txt"):
     include_package_data=True,
     extras_require={
         "coverage": [
-            "chanjo-report",
-            "pymysql",
+            "chanjo-report @ git+https://github.com/Clinical-Genomics/chanjo-report@remove_alchy"
```

Comment:
This is using this branch of chanjo-report that in turn uses [this branch](https://github.com/Clinical-Genomics/chanjo/pull/255) of chanjo